src/box-command.js

```diff
@@ -118,6 +118,36 @@ function getBooleanFlagValue(value) {
 	);
 }
 
+/**
+ * Removes all the undefined values from the object
+ *
+ * @param {Object} obj The object to format for display
+ * @returns {Object} The formatted object output
+ * @private
+ */
+
+function removeUndefinedValues(obj) {
```

Comment:
We have warning in Github Action Run: 
<img width="694" alt="image" src="https://github.com/user-attachments/assets/463048d8-c9af-4876-b99b-f4ea6d31ee52" />